cpp/tests/copying/copy_tests.cu

```diff
@@ -99,17 +101,27 @@ struct copy_if_else_tiny_grid_functor {
       // output
       std::unique_ptr<cudf::column> out = cudf::experimental::allocate_like(lhs, lhs.size(), cudf::experimental::mask_allocation_policy::RETAIN, mr);
 
-      // device views
-      auto lhs_dv = cudf::column_device_view::create(lhs);
-      auto rhs_dv = cudf::column_device_view::create(rhs);
+      // device views      
+      auto lhs_iter = cudf::make_pair_iterator<false, T>((T*)lhs.begin<T>(), static_cast<cudf::bitmask_type*>(nullptr));
```

Comment:
This may give invalid data in `lhs`  and `rhs` at null locations.

Reply:
Yes, but this is constructed for a specific test where nulls aren't passed (CopyIfElseTestTinyGrid)